### ISSUE TITLE
Only returning [statusCode] => 200

### ISSUE THREAD

**Issue by solasus** [fillup/walmart-partner-api-sdk-php#40]
Something odd is happening on my site.  Seems that I can only get an api response of 200.  Even on something as simple as an InventoryTest.  I am using the ENV_PROD and can confirm that my status changes to 401 if I change the consumerId = so I would assume that I'm authenticating.  

But something as simple as: 

```
$inventory = $client->get([
    'sku' => '1010868', // required
]);

echo 'inventory: '; print_r($inventory);
```

... should give me the quantity in stock of that (known) sku.   Any thoughts here...?

**Comment by fillup** [comment]
Yes that should return some data, can you share what the output of that script is?
